tsconfig.json

```diff
@@ -3,18 +3,21 @@
     "core/**/*",  // N.B.: also pulls in closure/goog/goog.js if needed.
     "closure/**/*",  // Just for ouptut directory structure.
   ],
+  "exclude": [
+    "core/blockly.js"
+  ],
   "compilerOptions": {
     // Tells TypeScript to read JS files, as
     // normally they are ignored as source files
     "allowJs": true,
 
     // Enable the next few options for type declarations.
     // Generate d.ts files
-    //"declaration": true,
+    "declaration": true,
     // Types should go into this directory.
     // Removing this would place the .d.ts files
     // next to the .js files
-    //"declarationDir": "build/ts/declarations",
+    "declarationDir": "build/ts/declarations",
```

Comment:
If we're not going to read this path from this file in `package_tasks.js`, I'd rather this be supplied via command line argument in the appropriate place in `buildJavaScript` in `build_tasks.js`, to avoid duplication.

Also: `ts/declarations/` is probably better than (the existing value of `TYPINGS_BUILD_DIR`) `typings/`, but unless you forsee generating some other files in the `ts/` subdirectory I'd go with `declarations/` or even just `d.ts/`.

Reply:
> but unless you forsee generating some other files in the `ts/` subdirectory

Hopefully soon enough we won't even have a `build/` directory! Webpack will just create a dist for us.